--- sqlx-sqlite-0.7.4/Cargo.toml	1970-01-01T00:00:01+00:00
+++ sqlx-sqlite-0.7.4/Cargo.toml	2024-07-07T08:40:24.250434+00:00
@@ -29,6 +29,9 @@
 
 [dependencies.chrono]
 version = "0.4.22"
+features = [
+    "clock",
+]
 optional = true
 default-features = false
 
@@ -65,11 +68,8 @@
 default-features = false
 
 [dependencies.libsqlite3-sys]
-version = "0.27.0"
+version = "0.28.0"
 features = [
-    "pkg-config",
-    "vcpkg",
-    "bundled",
     "unlock_notify",
 ]
 default-features = false
